2-completed-app/service/tableClient.js

```diff
@@ -0,0 +1,104 @@
+const { TableClient } = require("@azure/data-tables");
+const env = require("../configure/env")
+const serviceClient = TableClient.fromConnectionString(
+  env.connectionString,
+  env.tableName
+);
+
+const getData = async function(){
+    try{
+        const resArray = [];
+        const res = serviceClient.listEntities();
+    
+        for await(const entity of res){
+            resArray.push(entity)
+        }
+        return {code:0,data:resArray};
+    }catch(Error){
+        return {code:1,error:Error};
+    }
+ 
+}
+const filterData = async function(option){
```

Comment:
Can we document in JSDoc what the possible filters are and what options are available?